_posts/2020-10-02-contributing-for-the-first-time.adoc

```diff
@@ -0,0 +1,152 @@
+So don't be shy and reach out to us if you cant to contribute!
+
+In this article, I will try to give you some tips to help you create your first contribution to Quarkus.
+
+Before all, you must fork the Quarkus repository and build it on you local computer, even if you plan to contribute in an other area of Quarkus (more information on this later), you must be able to build the main Quarkus repository.
+
+Clone the repository : `git clone https://github.com/<your-fork-repo>/quarkus.git` then build Quarkus via `mvn clean install -Dquickly`.
+The Maven command will launch a minimal build of Quarkus and install the dependencies locally.
+
+If it didn't work follow the CONTRIBUTING.md (TODO link) page to setup your environment, anyway at some point you may need to read this page!
+
+If you're new to Quarkus: read the guides in the getting started page: https://quarkus.io/get-started/ (TODO link) you can skip the native and reactive ones if you don't plan to contribute on these areas,
+and as a new contributor I'll advise you to avoid them!
+
+You can read the CONTRIBUTE.md page for general guidelines, if you don't understand everything don't panic ;) or you can refer to it if needed as it can be hard to read for newcomers.
+
+Then, the hardest par will begin, you will need to find what you plan to do as a contributor, you can make contribution in the following areas:
+- documentation
+- code : core, extension, devtools
+- integration test
+- quickstart
+- the website
+
+Or just tell us your story with Quarkus, we live to hear user stories and have a dedicated page and blog category for them (TODO ADAPTER).
+
+We make our best to offer good first issues (TODO github link) and you can ask as many questions as you want (none are stupid questions) in our Zulip dev (TODO link) channel.
+We even setup a dedicated channel for Hacktoberfest: hackaton (TODO link).
+
+== Working on something
+
+When you're decided to work on something, you can find help on Zulip or Github by pinging area maintainers (TODO link).
+
+If there is an issue already opened, ask that someone assign it to you (you can add a comment for this on the issue).
+
+If you plan to work on new stuff, ask on Zulip or the mailing list before starting for all community to be aware of your plan.
+
+If you want to create a new extension, congratulation! But you should definitely ask on the mailing list plus create an extension proposal on github (TODO link).
+Extension contribution is a long process with a design phase, and a discussion phase prior to any implementation.
+As a newcomer, better to make small contributions first.
+
+How I have started? With small contributions: bugfixes and documentation.
+Then I had the opportunity to meet Quarkus core members Emmanuel Bernard and Clement Escoffier at Devoxx France and we had a long discussion on MongoDB followed by some email exchange.
```

Comment:
consider moving this "How I have started?" up front as a short intro and then go into the details.